src/module.ts

```diff
@@ -3,7 +3,7 @@ import { name, version } from "../package.json";
 import autoLitWrapper from "./runtime/plugins/autoLitWrapper";
 
 export interface NuxtSsrLitOptions {
-  litElementPrefix: string | string[];
+  litElementPrefix: string[];
```

Comment:
This is a breaking change. Can we accommodate a single string?

Reply:
Done!